arangod/GeneralServer/RestHandlerFactory.cpp

```diff
@@ -129,6 +129,8 @@ std::shared_ptr<RestHandler> RestHandlerFactory::createHandler(
 
 void RestHandlerFactory::addHandler(std::string const& path, create_fptr func,
                                     void* data) {
+  WRITE_LOCKER(locker, _lock);
```

Comment:
Oh je, war das ungeschützter Verkehr?

Reply:
It was fine before, because previously we added all handlers before the HTTP interface was opened.
So no other thread could concurrently access the list of handlers.